contrib/linearize/linearize-data.py

```diff
@@ -32,7 +32,7 @@ def uint32(x):
 
 def bytereverse(x):
 	return uint32(( ((x) << 24) | (((x) << 8) & 0x00ff0000) |
-		       (((x) >> 8) & 0x0000ff00) | ((x) >> 24) ))
+		(((x) >> 8) & 0x0000ff00) | ((x) >> 24) ))
```

Comment:
nack. There is no way we can force everyone to configure their editor in this specific way.

Reply:
I've now reverted this change and the corresponding rule (`E101: indentation contains mixed spaces and tabs`).